vectordraw.js

```diff
@@ -30,16 +31,25 @@ var VectorDraw = function(element_id, settings) {
 VectorDraw.prototype.template = _.template([
     '<div class="jxgboard" style="width:<%= width %>px; height:<%= height %>px;" />',
     '<div class="menu">',
-    '    <div>',
+    '    <div class="controls">',
     '        <select>',
     '        <% vectors.forEach(function(vec, idx) { %>',
     '            <option value="<%= idx %>"><%= vec.description %></option>',
     '        <% }) %>',
     '        </select>',
     '        <button class="add-vector">Add Selected Force</button>',
-    '        <button class="reset">Clear All</button>',
-    '        <button class="undo">Undo</button>',
-    '        <button class="redo">Redo</button>',
+    '        <button class="reset">Reset</button>',
+    '        <button class="undo" title="Undo"><span class="fa fa-undo" /></button>',
+    '        <button class="redo" title="redo"><span class="fa fa-repeat" /></button>',
+    '    </div>',
+    '    <div class="vector-properties">',
```

Comment:
@e-kolpakov http://tasks.opencraft.com/browse/OC-38 says to "Leave space below dropdown and buttons for "vector properties" div (we will display the vector length and angle there in a future task - mock it out with fake values for now)". 

The missing code will be added in another (future) task. I guess this is so that we can decide on the visual design before we actually make it work.